gateway/ln-gateway/proto/gatewaylnrpc.proto

```diff
@@ -96,6 +106,17 @@ message SubscribeInterceptHtlcsResponse {
   // A unique identifier for every intercepted HTLC
   // Used to identify an intercepted HTLC through processing and settlement
   bytes intercepted_htlc_id = 11;
+
+  // Optional CircuitKey that LND needs to complete the HTLC
+  CircuitKey key = 12;
```

Comment:
Can we generalize this? We should aim for a proto spec agnostic of LN implementation

Reply:
See comment above. I removed this structure but directly put the incoming channel id and the htlc id in the `SubscribeInterceptHtlcsResponse`.